### PR TITLE
[Load Test] Add playgame unit test

### PR DESCRIPTION
**Pull request by navalev**
### Issue: #111

 - [x] Tick here to confirm that you have run RunCodeFormatter.ps1

### Description:
Add PlayGame unit test for the following scenario:
1. user logs in
2. user checks leaderboard
3. user plays a level - simulated by random wait time
4. user posts the new score and checks the leaderboard

### Test:
run the load tests under NetherLoadTest project

### CODE REVIEW

File: NetherLoadTest/NetherLoadTest/NetherUnitTest.cs [hunk 1]
```diff
@@ -122,6 +122,25 @@ public void Init()
             TestContext.EndTimer("GetScore");
         }
 
+        [TestMethod]
+        public async Task playGame()
```
Comment (stuartleeks) [inline NetherLoadTest/NetherLoadTest/NetherUnitTest.cs]: Can you change the casing of the method name to `PlayGame`?

File: NetherLoadTest/NetherLoadTest/NetherUnitTest.cs [hunk 2]
```diff
@@ -122,6 +122,25 @@ public void Init()
...
+            // after a random wait time the user will post the new score and get the leaderboard again. 
+            await EnsureLoggedInAsync();
+
+            TestContext.BeginTimer("GetScore");
+            await _client.GetScoresAsync();
+            TestContext.EndTimer("GetScore");
+            // sleep between 30 seconds to 5 minutes
+            Thread.Sleep(s_random.Next(30, 300) * 1000);
+
+            TestContext.BeginTimer("PlayLevel");
```
Comment (stuartleeks) [inline NetherLoadTest/NetherLoadTest/NetherUnitTest.cs]: I think it's probably better to put separate times around the PostScore and GetScores here - will probably make it easier to see what it happening

File: NetherLoadTest/NetherLoadTest/NetherUnitTest.cs (hunk 1 above)
Comment (navalev) [inline NetherLoadTest/NetherLoadTest/NetherUnitTest.cs]: yes. done

**Comment by stuartleeks** [comment]
Changes look good from eye-balling the code... haven't pulled the code to run it yet.

**Pull request merged**